ex2/ex2.js

```diff
@@ -8,3 +8,25 @@
  *     fn(",,,", 4) === -1
  *     fn("hel,lowo,rld", 0) - Throws
  */
+function getTheCommaIndexAt(str, val) {
```

Comment:
val stands for value. While the parameter is a value, it stands for the commaIndex, comma, or nthComma (rename to one of these).